pkg/cmd/agent-task/capi/sessions_test.go

```diff
@@ -590,184 +466,30 @@ func TestListSessionsForRepo(t *testing.T) {
-					OwnerID:      10,
-					RepoID:       1000,
-					ResourceType: "pull",
-					ResourceID:   2000,
-					CreatedAt:    sampleDate,
-					PullRequest: &api.PullRequest{
-						ID:             "PR_node",
-						FullDatabaseID: "2000",
-						Number:         42,
-						Title:          "Improve docs",
-						State:          "OPEN",
-						IsDraft:        true,
-						URL:            "https://github.com/OWNER/REPO/pull/42",
-						Body:           "",
-						CreatedAt:      sampleDate,
-						UpdatedAt:      sampleDate,
-						Repository: &api.PRRepository{
-							NameWithOwner: "OWNER/REPO",
-						},
-					},
-					User: &api.GitHubUser{
-						Login:      "octocat",
-						Name:       "Octocat",
-						DatabaseID: 1,
-					},
-				},
-			},
-		},
-		{
-			// This happens at the early moments of a session lifecycle, before a PR is created and associated with it.
-			name:  "single session, no pull request resource",
-			limit: 10,
-			httpStubs: func(t *testing.T, reg *httpmock.Registry) {
-				reg.Register(
-					httpmock.WithHost(
-						httpmock.QueryMatcher("GET", "agents/sessions/nwo/OWNER/REPO", url.Values{
-							"page_number": {"1"},
-							"page_size":   {"50"},
-						}),
-						"api.githubcopilot.com",
-					),
-					httpmock.StringResponse(heredoc.Docf(`
-						{
-							"sessions": [
-								{
-									"id": "sess1",
-									"name": "Build artifacts",
-									"user_id": 1,
-									"agent_id": 2,
-									"logs": "",
-									"state": "completed",
-									"owner_id": 10,
-									"repo_id": 1000,
-									"resource_type": "",
-									"resource_id": 0,
-									"created_at": "%[1]s"
-								}
-							]
-						}`,
-						sampleDateString,
-					)),
-				)
-				// GraphQL hydration
-				reg.Register(
-					httpmock.GraphQL(`query FetchPRsAndUsersForAgentTaskSessions\b`),
-					httpmock.GraphQLQuery(heredoc.Docf(`
-						{
-							"data": {
-								"nodes": [
-									{
-										"__typename": "User",
-										"login": "octocat",
-										"name": "Octocat",
-										"databaseId": 1
-									}
-								]
-							}
-						}`,
-						sampleDateString,
-					), func(q string, vars map[string]interface{}) {
-						assert.Equal(t, []interface{}{"U_kgAB"}, vars["ids"])
-					}),
-				)
-			},
-			wantOut: []*Session{
-				{
 
-					ID:           "sess1",
-					Name:         "Build artifacts",
-					UserID:       1,
-					AgentID:      2,
-					Logs:         "",
-					State:        "completed",
-					OwnerID:      10,
-					RepoID:       1000,
-					ResourceType: "",
-					ResourceID:   0,
-					CreatedAt:    sampleDate,
-					User: &api.GitHubUser{
-						Login:      "octocat",
-						Name:       "Octocat",
-						DatabaseID: 1,
-					},
-				},
-			},
-		},
-		{
-			name:    "multiple sessions, paginated",
-			perPage: 1, // to enforce pagination
-			limit:   2,
-			httpStubs: func(t *testing.T, reg *httpmock.Registry) {
+				// Page 2 returns older duplicate sessions for 3000 and 3001, plus another new PR 3002
```

Comment:
There's only a dup for 3000 in this test case, so the comment should be updated.